src/participants.js

```diff
@@ -20,10 +20,36 @@ export const calculateNumAttended = participants => participants.reduce((m, v) =
   return m + (attended ? 1 : 0)
 }, 0)
 
-export const calculateFinalizeMaps = participants => {
+export const calculateFinalizeMaps = (participants, overrideMissingValue = false) => {
```

Comment:
I was thinking about the following 3 options

- `calculateFinalizeMaps(participants)` = Default, raises error if there is a missing value
- `calculateFinalizeMaps(participants, overrideMissingValue=1)` = set missing value as 1
- `calculateFinalizeMaps(participants, overrideMissingValue=0)`= set missing value as 0
- `calculateFinalizeMaps(participants, overrideMissingValue=12345)`= invalid value, throw error

Reply:
Would be better to have the caller pass in the `PARTICIPANT_STATUS.<value>` as the `overrideMissingValue`.

Reply:
@hiddentao not sure about that. PARTICIPANT_STATUS has 5 valid statuses but we only have 3 (1, 0, null=throw error) and they don't map equally

Reply:
Just disallow the ones that don't map then.

Reply:
Otherwise, create a new enum to pass in as that would still be better than passing in magic numbers.